easybuild/framework/easyconfig/default.py

```diff
@@ -130,6 +130,7 @@
     'sanity_check_paths': [{}, ("List of files and directories to check "
                                 "(format: {'files':<list>, 'dirs':<list>})"), BUILD],
     'skip': [False, "Skip existing software", BUILD],
+    'skip_mod_files_check': [False, "Skip the check for .mod files in a GCCcore level install", BUILD],
```

Comment:
maybe rename to `skip_mod_files_sanity_check`?

Reply:
done in 0b3daf2